pkgs/applications/misc/gnuradio-ais/default.nix

```diff
@@ -26,11 +26,15 @@ stdenv.mkDerivation rec {
     done
   '';
 
+  patches = [ ./fix-volk.patch ];
```

Comment:
Unless you have an objection, I'm going to change the `rev` to point to my PR commit, since it removes the need to include the patch here